mobile/src/main/java/org/openhab/habdroid/util/ExtensionFuncs.kt

```diff
@@ -226,3 +228,12 @@ fun Context.showToast(@StringRes message: Int) {
 fun Context.hasPermission(permission: String): Boolean {
     return ContextCompat.checkSelfPermission(this, permission) != PackageManager.PERMISSION_GRANTED
 }
+
+fun Activity.finishAndRemoveTaskIfPossible() {
```

Comment:
Why as extension function? A `protected` fun in `AbstractBaseActivity` should be fine.

Reply:
Except `NfcReceiveActivity` doesn't derive from that and also doesn't need to when doing this as extension function ;-)

Reply:
Ah, I didn't see that.